docs/self-hosted/install/configure-ingress.md

```diff
@@ -49,30 +17,34 @@ Installing Gitpod on a subdomain works as well. For example:
     *.gitpod.your-domain.com
     *.ws.gitpod.your-domain.com
 
-### HTTPS
-While we highly recommend operating Gitpod using HTTPS, Gitpod is able to run on insecure HTTP.
-If you use Gitpod's internal Docker registry, the downside of not using HTTPS is that Kubernetes won't be able to pull images from the registry because it considers the registry insecure.
-You can either resort to using an [external registry](#docker-registry-optional) or use HTTPS. For running Gitpod on insecure HTTP, no HTTPS certificates are needed and you can skip this section.
+# HTTPS
+Gitpod requires HTTPS certificates to function properly. We recommend using [Let's Encrypt](https://letsencrypt.org/) for retrieving certificates as we do for [gitpod.io](https://gitpod.io).
 
 > Important: The HTTPS certificates for your domain must include `your-domain.com`, `*.your-domain.com` and `*.ws.your-domain.com`. Beware that wildcard certificates are valid for one level only (i.e. `*.a.com` is not valid for `c.b.a.com`).
 
-To use the HTTPS certificates for your domain
- - `echo values/https.yaml >> configuration.txt`
- - place your certificates in `secrets/https-certificates/` like so:
+To configure the HTTPS certificates for your domain
+ 1. [Generate certificates](#using-let-s-encrypt) and put your certificate files under `secrets/https-certificates/`:
```

Comment:
Don't users need to create a secret via `kubectl create secret`? 
I think our helm chart will only pick up secrets from `secrets/https-certificates/` if that folder is inside a local copy of our chart. I assume we don't encourage users to make such copies and encourage users instead to pull the chart from charts.gitpod.io.

Reply:
> Don't users need to create a secret via kubectl create secret?

Good point, did not think about that yesterday but just re-wrote the existing stuff.

> and encourage users instead to pull the chart from charts.gitpod.io.

That'd be a nice goal indeed. Will check if that's easily achievable.

Reply:
Found a situation where it does not work that well, at least: When using a custom DB you want to execute the DB scripts, where it is handy to have them locally.